buildutils/src/ensure-repo.ts

```diff
@@ -304,7 +304,7 @@ export async function ensureIntegrity(): Promise<boolean> {
         return;
       }
       const depData = graph.getNodeData(depName);
-      if (depData.style) {
+      if (typeof depData.style === 'string') {
```

Comment:
This commit is full of small logic changes (changes to compiled JS) that avoids taking certain actions if some of the vars are missing (undefined/null), or in other words "failing" silently. Some of these might not be wanted (i.e. we want a noisy failure and bug report). For such cases we would normally go from `?.` to `!.` (assert that it is non-nullable, and have the code throw if it turns out to be the case).

Reply:
I think it is fair to stick with `?.` if possible and deal with missing data.